ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -764,13 +764,17 @@ export default class ConfirmTransactionBase extends Component {
       ({ id }) => id === selectedInsightSnapId,
     );
 
+    const allowedTransactionTypes =
+      txData.type === TRANSACTION_TYPES.CONTRACT_INTERACTION ||
+      txData.type === TRANSACTION_TYPES.SIMPLE_SEND ||
+      txData.type === TRANSACTION_TYPES.TOKEN_METHOD_SAFE_TRANSFER_FROM ||
```

Comment:
Did you test all of these transaction types?

Reply:
Yes !